src/scss/login.module.scss

```diff
@@ -133,6 +143,23 @@ $white: #fff;
   opacity: 0.8;
 }
 
+@media only screen and (max-width: 561px) {
+  .login-github {
+    font-size: 1rem;
+  }
+}
+@media only screen and (max-width: 470px) {
```

Comment:
Can we remove the margin adjustments, and just center the content?

Reply:
I did try to achieve that and make the css simpler, but the div structure isn't exactly hierarchical so it was kind of a hassle. We can fix the hacks when we rewrite the landing maybe?

Reply:
Increasing the `margin-top` to `9rem` solved the issue. It maybe due to the placement of `MLH` beside the scrollbar, which was adding an offset to `Login with github` button.

It should be centred now, after adding the `9rem` fix, could you check?

> Also, mark the conversation as unresolved, if you have left any questions on it, becomes much more visible. To avoid opening resolved conversations to check any updates in it

Reply:
Ah, it does work now. 
The margins were moving the element diagonally but now setting top margin only to 9rem seems to have fixed it, do you know why that was the case earlier and why both left and top margins were used?

Reply:
> Increasing the `margin-top` to `9rem` solved the issue. It maybe due to the placement of `MLH` beside the scrollbar, which was adding an offset to `Login with github` button.

I guess it was due to the `MLH`, but don't quote me on that. Earlier the button was placed in the `ellipse` class, hence the manual `left-margin` was added to compensate the scrollbar offset.

But now as the `github button` is moved further down, the scrollbar offset is not a problem anymore